go/teams/service_helper.go

```diff
@@ -336,6 +371,19 @@ func RemoveMember(ctx context.Context, g *libkb.GlobalContext, teamname, usernam
 	return t.ChangeMembership(ctx, req)
 }
 
+func CancelEmailInvite(ctx context.Context, g *libkb.GlobalContext, teamname, email string) error {
+	t, err := GetForTeamManagementByStringName(ctx, g, teamname, true)
+	if err != nil {
+		return err
+	}
+
+	if !libkb.CheckEmail.F(email) {
+		return errors.New("Invalid email address")
+	}
+
+	return removeMemberInvite(ctx, g, t, email, keybase1.UserVersion{})
```

Comment:
It seems yucky to know it's an email here, and then have to reparse that it's in email in `removeMemberInvite`.  Can you pass in `typ` to `removeMemberInvite`?

Reply:
Ok, I'll split that function up.  Figuring out the type is still needed for non-email removes, but I'll make a new function to handle remove after type is figured out.

Reply:
ok thx

Reply:
done